vertexai/src/main/java/vertexai/gemini/samples/ChatDiscussion.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package vertexai.gemini.samples;
+
+import com.google.cloud.vertexai.VertexAI;
+import com.google.cloud.vertexai.generativeai.preview.ChatSession;
+import com.google.cloud.vertexai.generativeai.preview.GenerativeModel;
+import com.google.cloud.vertexai.generativeai.preview.ResponseHandler;
+import com.google.cloud.vertexai.v1beta1.GenerateContentResponse;
```

Comment:
Sorry for the late notice: I am planning to change the package so that all things from `com.google.cloud.vertexai.v1beta1` are from `com.google.cloud.vertexai.api` 

Context: we found some discrepancy between the v1 and v1beta1, which means if we later do the version shift, the user may need to do more things than changing all "v1beta1" to "v1", so I'm hoping to hide the version completely and treat every version change as implementation detail.

Reply:
Ok, good to know.
I'll leave it as is till there's a newer drop with the mentioned package change.